packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -0,0 +1,240 @@
+# Cadl Ranch Project summary
+
+### MultiInterfaceClient_dogs_getDogs
+
+Illustrate grouping operations on subclient.
```

Comment:
Would it be possible to parse and split this into

```md
## MultiInterfaceClient
...
### dogs_getDogs
```

Not sure if this is better, but just floating the idea

Reply:
Did think about this but I think markdown is not the best at showing the hierarchy.

Reply:
maybe what could look better is if of the title have a link to where the scenario is defined.

Reply:
oh yeah that would be really cool